CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.9.2 (Next)
 
 * [#161](https://github.com/slack-ruby/slack-ruby-client/pull/161): Added support for cursor pagination - [@dblock](https://github.com/dblock).
+* [#162](https://github.com/slack-ruby/slack-ruby-client/pull/162): Gracefully close websocket on errno::epipe - [@johanoskarsson](https://github.com/johanoskarsson).
```

Comment:
Can I be difficult and nitpick? The correct class is `Errno::EPIPE`, it's case sensitive, so quote it as code and correct upper/lowercase please.